specification/baremetalinfrastructure/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+```
+
+### Tag: package-2020-08-06-preview and go
+
+These settings apply only when `--tag=package-2020-08-06-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2020-08-06-preview' && $(go)
+namespace: Microsoft.BareMetalInfrastructure
```

Comment:
change go SDK namespace to lower case. And looking across other go SDK namespaces, they are all single word without Microsoft.